src/coprocessor/dag/executor/topn.rs

```diff
@@ -120,6 +121,19 @@ impl Executor for TopNExecutor {
             None => Ok(None),
         }
```

Comment:
(Unless you need some logging this whole thing just equivalent to `Ok(iter.next())`, but since this is old code I'm fine with not modifying it)

Reply:
addressed, It should be corrected, thanks.